Ports/lzop/patches/ReadMe.md

```diff
@@ -0,0 +1,12 @@
+# Patches for lzop on SerenityOS
+
+## `0001-Don-t-try-to-use-malloc.h-header-file-in-the-codebas.patch`
+
+Don't try to use <malloc.h> header file in the codebase
+
+
+## `0002-Don-t-try-to-use-malloc.h-header-file-in-miniacc.h.patch`
+
+Don't try to use <malloc.h> header file in miniacc.h
+
```

Comment:
Aren't these basically the same, with `miniacc.h` being a subset of "the codebase"? They are certainly fixing the same issue, so I'd put them into the same patch.

Reply:
I see your point here, I'll merge these patches to one patch.